extensions/common/http/jetty-core/src/main/java/org/eclipse/edc/web/jetty/JettyService.java

```diff
@@ -203,6 +205,7 @@ private void configure(ServerConnector connector) {
     @NotNull
     private HttpConnectionFactory httpConnectionFactory() {
         HttpConfiguration https = new HttpConfiguration();
+        https.setSendServerVersion(SEND_SERVER_VERSION);
```

Comment:
Please set this to false inline instead of using a static constant, as it is easier to read. Also, prefer primitives.

Reply:
@jimmarino I have adjusted it.